mm2src/coins/z_coin.rs

```diff
@@ -826,42 +883,19 @@ impl<'a> UtxoCoinBuilder for ZCoinBuilder<'a> {
         .expect("DEX_FEE_Z_ADDR is a valid z-address")
         .expect("DEX_FEE_Z_ADDR is a valid z-address");
 
-        let params_dir = match &self.z_coin_params.zcash_params_path {
-            None => default_params_folder().or_mm_err(|| ZCoinBuildError::ZCashParamsNotFound)?,
-            Some(file_path) => PathBuf::from(file_path),
-        };
-
-        let z_tx_prover = async_blocking(move || {
-            let (spend_path, output_path) = get_spend_output_paths(params_dir)?;
-            let verification_successful = verify_checksum_zcash_params(&spend_path, &output_path)?;
-            if verification_successful {
-                Ok(LocalTxProver::new(&spend_path, &output_path))
-            } else {
-                MmError::err(ZCoinBuildError::SaplingParamsInvalidChecksum)
-            }
-        })
-        .await?;
-
         let my_z_addr_encoded = encode_payment_address(
             self.protocol_info.consensus_params.hrp_sapling_payment_address(),
             &my_z_addr,
         );
 
+        let blocks_db = self.blocks_db().await?;
         let evk = ExtendedFullViewingKey::from(&z_spending_key);
```

Comment:
Same here, do you have a reason to move blocks_db initialization above `ExtendedFullViewingKey`?

Reply:
`evk` was used to initialize `walletdb` before in builder but now, `walletdb mod` handles this.. 

so `evk` is never used again actually, I'll delete the line.